scripts/mktestdb.py

```diff
@@ -3,22 +3,27 @@
 import sys
 import logging
 from argparse import ArgumentParser
-from pkg_resources import resource_filename
 
 from sqlalchemy import create_engine
 
 from pycds.util import create_test_database, create_test_data
-    
+
 if __name__ == '__main__':
     parser = ArgumentParser(description="Script to create a test CRMP database and write it to a test database. DSN strings are of form:\n\tdialect+driver://username:password@host:port/database\nExamples:\n\tpostgresql://scott:tiger@localhost/mydatabase\n\tpostgresql+psycopg2://scott:tiger@localhost/mydatabase\n\tpostgresql+pg8000://scott:tiger@localhost/mydatabase")
-    parser.add_argument("-d", "--dsn", help="Source database DSN from which to read ")
-    parser.add_argument("-t", "--testdsn", help="Destination DSN to write to")
+    parser.add_argument("-t",
+                        "--testdsn",
+                        help="Destination DSN to write to")
+    parser.add_argument("-i",
+                        "--include-data",
+                        action='store_true',
+                        help="Include test data in build")
```

Comment:
In the "build"? I wouldn't call a database a "build". Maybe: "Create a small test dataset in the new database".

Reply:
Sounds much better, I have changed it and pushed a new commit.